sources/packages/web/src/views/student/StudentApplicationDetails.vue

```diff
@@ -136,6 +148,9 @@ export default {
         {
           label: "View",
           icon: "pi pi-fw pi-folder-open",
+          command: () => {
```

Comment:
This could be simplified to:
```
command: viewApplicaion
```

Reply:
just copied from edit application :) will change both to sync